examples/todo-typescript/src/server/actions.ts

```diff
@@ -0,0 +1,33 @@
+import HttpError from '@wasp/core/HttpError.js';
+import { Context, Task } from './serverTypes'
+
+type CreateArgs = { description: Task['description'] };
+
+export async function createTask({ description }: CreateArgs, context: Context) {
+  if (!context.user) {
+    throw new HttpError(401);
+  }
+
+  return context.entities.Task.create({
+    data: {
+      description,
+      user: { connect: { id: context.user.id } },
+    },
+  });
+};
+
+type UpdateArgs = { taskId: Task['id']; isDone: Task['isDone'] };
```

Comment:
If you changed `taskId` to `id`, you could also simplify this with `Pick`.

Reply:
🚀